src/main/java/frc/trigon/robot/subsystems/gripper/GripperConstants.java

```diff
@@ -0,0 +1,221 @@
+            ARM_LENGTH_METERS,
+            ARM_MASS_KILOGRAMS,
+            ARM_MINIMUM_ANGLE,
+            ARM_MAXIMUM_ANGLE,
+            SHOULD_SIMULATE_GRAVITY
+    );
+
+    static final SysIdRoutine.Config SYSID_CONFIG = new SysIdRoutine.Config(
+            Units.Volts.of(1.5).per(Units.Seconds),
+            Units.Volts.of(2),
+            Units.Second.of(1000)
+    );
+
+    private static final double MAXIMUM_DISPLAYABLE_VELOCITY = 12;
+    static final SpeedMechanism2d GRIPPER_MECHANISM = new SpeedMechanism2d(
+            "GripperMechanism",
+            MAXIMUM_DISPLAYABLE_VELOCITY
+    );
+    static final SingleJointedArmMechanism2d GRIPPER_ANGLE_MECHANISM = new SingleJointedArmMechanism2d(
+            "GripperAngleMechanism",
+            ARM_LENGTH_METERS,
+            Color.kRed
+    );
+
+    static {
+        configureGripperMotor();
+        configureAngleMotor();
+        configureEncoder();
+        configureLaserCAN();
+    }
+
+    private static void configureGripperMotor() {
+        final TalonFXConfiguration config = new TalonFXConfiguration();
+
+        config.Audio.BeepOnBoot = false;
+        config.Audio.BeepOnConfig = false;
+
+        config.MotorOutput.Inverted = GRIPPER_MOTOR_INVERTED_VALUE;
+        config.MotorOutput.NeutralMode = GRIPPER_MOTOR_NEUTRAL_MODE_VALUE;
+        config.Feedback.RotorToSensorRatio = GRIPPER_MOTOR_GEAR_RATIO;
+
+        GRIPPER_MOTOR.applyConfiguration(config);
+        GRIPPER_MOTOR.setPhysicsSimulation(GRIPPER_SIMULATION);
+
+        GRIPPER_MOTOR.registerSignal(TalonFXSignal.STATOR_CURRENT, 100);
+        GRIPPER_MOTOR.registerSignal(TalonFXSignal.MOTOR_VOLTAGE, 100);
+    }
+
+    private static void configureAngleMotor() {
+        final TalonFXConfiguration config = new TalonFXConfiguration();
+
+        config.Audio.BeepOnBoot = false;
+        config.Audio.BeepOnConfig = false;
+
+        config.MotorOutput.Inverted = ANGLE_MOTOR_INVERTED_VALUE;
+        config.MotorOutput.NeutralMode = ANGLE_MOTOR_NEUTRAL_MODE_VALUE;
+        config.Feedback.RotorToSensorRatio = ANGLE_MOTOR_GEAR_RATIO;
+
+        config.Slot0.kP = ANGLE_P;
+        config.Slot0.kI = ANGLE_I;
+        config.Slot0.kD = ANGLE_D;
+        config.Slot0.kS = ANGLE_KS;
+        config.Slot0.kA = ANGLE_KA;
+        config.Slot0.kV = ANGLE_KV;
+        config.Slot0.kG = ANGLE_KG;
+
+        config.Feedback.FeedbackRemoteSensorID = GRIPPER_ANGLE_ENCODER_ID;
+        config.Feedback.FeedbackSensorSource = ANGLE_ENCODER_TYPE;
+
+        ANGLE_MOTOR.applyConfiguration(config);
+        ANGLE_MOTOR.setPhysicsSimulation(ARM_SIMULATION);
+
+        ANGLE_MOTOR.registerSignal(TalonFXSignal.ROTOR_POSITION, 100);
+        ANGLE_MOTOR.registerSignal(TalonFXSignal.MOTOR_VOLTAGE, 100);
+        ANGLE_MOTOR.registerSignal(TalonFXSignal.POSITION, 100);
+        ANGLE_MOTOR.registerSignal(TalonFXSignal.VELOCITY, 100);
+        ANGLE_MOTOR.registerSignal(TalonFXSignal.CLOSED_LOOP_REFERENCE, 100);
+    }
```

Comment:
you're missing some motion magic settings 
make sure to read the motion magic docs